example/config/managedk8s.yaml

```
@@ -26,11 +26,10 @@ providers:                   # contains information about known providers
     #       kind: "Pod"
     #       matchLabels:
     #         foo: bar
-    #       # Defaults to ["default", "kube-public", "kube-node-lease"]
-    #       namespaceNames:
-    #       - default
-    #       - kube-public
-    #       - kube-node-lease
+    #       # only pods in namespaces ["default", "kube-public", "kube-node-lease"] are meaningful to be selected with namespaceMatchLabels
+    #       # since the rule does not perform checks on objects in namespaces different from the listed above
+    #       namespaceMatchLabels:
+    #          kubernetes.io/metadata.name: default
```

You have an extra whitespace here.
```suggestion
    #         kubernetes.io/metadata.name: default
```